sites/all/themes/pratchett/template.php

```diff
@@ -105,3 +105,29 @@ function pratchett_ting_object_cover($variables) {
 
   return '<div class="ting-cover-wrapper">' . $output . $icons . '</div>';
 }
+
+/**
+ * Implements hook_preprocess_menu_link().
+ *
+ *  Add destination to login link when using SSO authentication.
+ */
+function pratchett_preprocess_menu_link(&$variables) {
+  if (module_exists('ding_adgangsplatformen')) {
+    $element = &$variables['element'];
+    if ($element['#href'] == DING_ADGANGSPLATFORMEN_LOGIN_URL) {
+      $destination = drupal_get_destination();
+      // Handle issues with lazy-pane and destination.
+      if ($destination['destination'] === 'lazy-pane/ajax') {
+        $path = $_GET['q'];
```

Comment:
Can we use `current_path()` here or is this run too early in the bootstrap process?

Reply:
No, this is from ddb-core and current_path did not work here. Lazy panel do some strang stuff.